### PR TITLE
fix parsing of anisotropic B factors

### PR DESCRIPTION
**Pull request by rtf-const**
This pull request addresses issue #1733 

- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file and understand that AppVeyor and
TravisCI will be used to confirm the Biopython unit tests and ``flake8`` style
checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

### CODE REVIEW

File: Tests/test_PDB_MMCIFParser.py [hunk 1]
```diff
@@ -276,12 +326,12 @@ def test_conversion(self):
         self.assertEqual(len(pdb_struct), len(cif_struct))
 
         pdb_atom_names = [a.name for a in pdb_struct.get_atoms()]
-        cif_atom_names = [a.name for a in pdb_struct.get_atoms()]
+        cif_atom_names = [a.name for a in cif_struct.get_atoms()]
```
Comment (rtf-const) [inline Tests/test_PDB_MMCIFParser.py]: I know that it is unrelated to the issue, but it obviously looks as typo error

Comment (peterjc) [reply Tests/test_PDB_MMCIFParser.py]: It does look like a typo - I'd happily merge this fix (and the matching change later in ``testModels``) as a separate PR while we wait for a review of the main changes proposed on #1775.

**Comment by jgreener64** [comment]
This looks alright to me.

**Comment by peterjc** [comment]
Thanks Joe and Konstantin, merging...

**Pull request merged**